_dev/src/components/configuration/facebook-connected.vue

```diff
@@ -283,7 +239,6 @@ import {
   BDropdownItem,
 } from 'bootstrap-vue';
 import FacebookApp from './facebook-app.vue';
-import facebookLogo from '../../assets/facebook_logo.svg';
```

Comment:
Is this working from the storybook too ?

Reply:
Indeed

![image](https://user-images.githubusercontent.com/6768917/116987407-f04cd000-acc6-11eb-8387-328a43fd0ab8.png)

Plus, it was already modified on the non-connected version.